app/controllers/payment-links/metadata/metadata-form.js

```diff
@@ -80,4 +105,29 @@ function isNotEmpty (value) {
   return value && value.length !== 0
 }
 
+function isValidLengthForColumnHeader (value) {
+  return value && value.length !== 0 && value.length <= METADATA_MAX_HEADER_LENGTH
+}
+
+function isValidLengthForCellContent (value) {
```

Comment:
Ended up splitting up the `isValidLength` method into:

`isValidLengthForColumnHeader ` & `isValidLengthForCellContent `